osf/models/mixins.py

```diff
@@ -315,6 +320,13 @@ def add_affiliated_institution(self, inst, user, save=False, log=True):
                 auth=Auth(user)
             )
 
+        if save:
+            self.save()
+
+            self.update_search()
+            return True
```

Comment:
I see we have a `save` parameter. However, my understanding is relationship update doesn't need `save()`. In addition, no use case of this `save` parameter as `True` in the code base either. Finally, if nothing is changed, why calling `.update_search()`?

Reply:
It's no it's necessary, but if note the function original signature the save is getting passed in, it seemed safer to have that actually do something as opposed to remove the unused save parameter.

Reply:
Can you do a quick scan of this again, if no place is using this save (`save=True`), we can remove it. If there are cases using the default value (`save=False`), we can remove that too.

Reply:
Ok I looked everywhere and it’s good.

Reply:
Save was set to true but it wasn’t necessary as it didn’t do anything.